components/create_comment/create_comment.jsx

```diff
@@ -184,6 +184,16 @@ class CreateComment extends React.PureComponent {
          * The last time, if any, when the selected post changed. Will be 0 if no post selected.
          */
         selectedPostFocussedAt: PropTypes.number.isRequired,
+
+        /**
+         * Update preview for textbox
+         */
+        updatePreview: PropTypes.func.isRequired,
```

Comment:
This would become `setShowPreview` in the above. (I think it would probably be overkill to pass in a `showPreview` and a `hidePreview` here, but 0/5.)